src/scss/common.scss

```
@@ -0,0 +1,37 @@
+$common-styles: (
+  "https://sf.gov/core/themes/stable/css/system/components/ajax-progress.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/align.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/autocomplete-loading.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/fieldgroup.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/container-inline.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/clearfix.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/details.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/hidden.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/item-list.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/js.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/nowrap.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/position-container.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/progress.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/reset-appearance.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/resize.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/sticky-header.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/system-status-counter.css",
+  "https://sf.gov/core/themes/stable/css/system/components/system-status-report-counters.css",
+  "https://sf.gov/core/themes/stable/css/system/components/system-status-report-general-info.css",
+  "https://sf.gov/core/themes/stable/css/system/components/tabledrag.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/tablesort.module.css",
+  "https://sf.gov/core/themes/stable/css/system/components/tree-child.module.css",
+  "https://sf.gov/core/themes/stable/css/views/views.module.css",
+  "https://sf.gov/modules/contrib/extlink/extlink.css",
+  "https://sf.gov/modules/contrib/paragraphs/css/paragraphs.unpublished.css",
+  "https://sf.gov/libraries/sfgov-pattern-lab/dist/css/components.css",
+  "https://sf.gov/libraries/sf-design-system/css/all.css",
+  "https://sf.gov/themes/custom/sfgovpl/dist/css/drupal.css",
+  "https://sf.gov//fonts.googleapis.com/css?family=Rubik:300,400,500,700",
+  "https://sf.gov//fonts.googleapis.com/css?family=Noto+Sans+TC:300,400,500,700&amp;display=swap&amp;subset=chinese-traditional",
```

Sorry, this was my fault! 🤦‍♀️ 

```suggestion
  "https://fonts.googleapis.com/css?family=Rubik:300,400,500,700",
  "https://fonts.googleapis.com/css?family=Noto+Sans+TC:300,400,500,700&amp;display=swap&amp;subset=chinese-traditional",
```
```